rye/src/cli/build.rs

```diff
@@ -75,14 +78,23 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
             style(project.normalized_name()?).cyan()
         );
 
-        let mut build_cmd = Command::new(get_venv_python_bin(&venv));
+        let mut build_cmd = Command::new(get_venv_python_bin(&self_venv));
         build_cmd
             .arg("-mbuild")
             .env("NO_COLOR", "1")
             .arg("--outdir")
             .arg(&out)
             .arg(&*project.root_path());
 
+        if use_uv {
+            // we need to ensure uv is available to use without installing it into self_venv
+            let uv = UvBuilder::new().with_output(output).ensure_exists()?;
+            build_cmd.arg("--installer=uv");
+            build_cmd.env("PATH", uv.uv_bin().parent().ok_or_else(||
+                anyhow!("Could not find uv binary in self venv: empty path")
+            )?);
```

Comment:
This PATH upgrade needs review or comments before this is done. In this shape, it works for me. And it's faster too. :slightly_smiling_face:

Reply:
The question is, is it needed to expose the rest of the PATH variable to the build?

Reply:
Now I've gone with prepend to $PATH as the solution instead, which should be the normal kind of solution.